python/py-nbconvert/Portfile

```diff
@@ -4,14 +4,14 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-nbconvert
-version             7.2.1
-revision            1
+version             7.2.9
+revision            0
 categories-append   textproc
 license             BSD
 supported_archs     noarch
 platforms           {darwin any}
 
-python.versions     37 38 39 310 311
+python.versions     37 38 39 310
```

Comment:
why are you removing the py311 subport?

Reply:
Sorry, I had prepared this pull request from an outdated Portfile. Meanwhile, the py311 subport has been added by https://github.com/macports/macports-ports/commit/f3b2e2f8ac97c9a8930ad74f22aaff34246367de .